dotenv/testdata/.env.test

```diff
@@ -5,6 +5,7 @@ AFTER_EMPTY=empty
 EMPTY_VALUE=
 QUOTED_SINGLE='single quoted'
 QUOTED_DOUBLE="double quoted"
+EMPTY_QUOTED=""
```

Comment:
Should we include a test for empty single quotes as well?

Reply:
I don't think it's strictly necessary but having one more test to prevent regressions will not hurt.

Reply:
It turns out it was strictly necessary, there was a bug with single quote parsing. Nice catch!